.vscode/settings.json

```diff
@@ -0,0 +1,55 @@
+﻿{
```

Comment:
This file must not be uploaded, add the whole .vscode folder to .gitignore please!!